src/main/java/edu/kis/vh/nursery/DefaultCountingOutRyhmer.java

```diff
@@ -0,0 +1,43 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRyhmer {
+
+    private static final int NUMBERS_CAPACITY = 12;
+    private static final int DEFAULT_EMPTY_STACK_VALUE = -1;
+    private static final int DEFAULT_FULL_STACK_VALUE = 11;
```

Comment:
This constant is related to NUMBERS_CAPACITY . Changing only one leads to an error (inconsistency).